src/test/java/org/broadinstitute/hellbender/tools/spark/sv/utils/SVUtilsUnitTest.java

```diff
@@ -4,12 +4,9 @@
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
-/**
- * Unit tests for SVUtils.
- */
 public class SVUtilsUnitTest extends BaseTest {
```

Comment:
After https://github.com/broadinstitute/gatk/pull/3475, this should extend `GATKBaseTest`